### ISSUE TITLE
Rinsing?

### ISSUE THREAD

**Issue by sr320** [RobertsLab/project-pacific.oyster-larvae#6]
As far as rinsing goes, should I be using ethanol? Or freshwater rinse, dry the seed, flash freeze and store at -80C? This is what we did last year (minus the freshwater rinse) with the oyster seed. I know we talked recently about not flash freezing for the larvae, but it was taking me about 30 minutes to do all the rinsing and sampling with the geoducks in the greenhouse. Because the -80C is a hop skip and a jump away, I couldn’t really just stick them in the -80C after every sample taken or I’d probably just give in to sleeping at the hatchery. So the earlier samples I collected sat for up to 30 minutes in an icebath until I was done collecting all the samples. Benoit and I will probably discuss getting liquid nitrogen.

**Comment by sr320** [comment]
Will bring Emma in- but have been talking about snap freezing in TCA. Again a detailed protocol is necessary here.

**Comment by emmats** [comment]
Do not rinse in ethanol!
The FW rinse, flash freezing, and storing at -80 sounds the best.  
If samples are sitting for a prolonged time in an ice bath then you will probably get a proteomic response to the cold, so that is not ideal.
You can also flash freeze with ethanol, super cooled with dry ice.

**Comment by sr320** [comment]
@emmats What about TCA? Should we not be using this? That is what we are doing with wild larvae samples.

**Comment by emmats** [comment]
The TCA is fine, but I'm still not sure how well it will work with a shelled larva.  It needs to permeate tissue quickly to really work effectively.  It can be added as a safe guard, though.  Part of the reason we use it with bacteria is because they don't necessarily die right away at very cold temperatures, whereas most more complex organisms are going to cease cellular activity at -80 and lower.

In summary: go for the TCA, but don't skip the flash freezing.

**Comment by Ellior2** [comment]
Ok, sounds like we will have to get our hands on some of this TCA. I had put the samples in an icebath while sampling since it was taking so long and it was 80F in the greenhouse, but I can now see why that wasn't ideal. Flash freezing seems to be the way to go. The liquid nitrogen worked really well for us and we had plenty left over after the experiment so we will probably go with that. We have a good dewar so it would just be a matter of picking it up. Dry ice is a pain because the source is unreliable in Port Townsend and it doesn't last as long.
